docker_images/diffusers/app/lora.py

```diff
@@ -101,9 +101,11 @@ def _load_textual_embeddings(self, adapter, model_data):
         if self._is_pivotal_tuning_lora(model_data):
             embedding_path = self._hub_repo_file(
                 repo_id=adapter,
-                filename="embeddings.safetensors"
-                if self._is_safetensors_pivotal(model_data)
-                else "embeddings.pti",
+                filename=(
+                    "embeddings.safetensors"
+                    if self._is_safetensors_pivotal(model_data)
+                    else "embeddings.pti"
+                ),
```

Comment:
Seems unrelated?

Reply:
yes -> `black .` reformatting made this on my laptop, removing, probably a lib version difference